.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: python
+
+python:
+- '2.7'
+
+install:
+- pip install -r requirements.txt
+
+script:
+- python -m unittest discover -p '*test*.py'
+
+#notifications:
+#  slack:
+#    secure: tYNwi4kff+8FWbv+nWqKZt28Qez0Dv77rJG6nvdiaO7QgPZmaiG5l8NqYdxV63CMgNzHwTFAz8ca9uOoh06o7+HuiFNNiqXEAu8umLvlnvkKT6itSDyImxxD7ypitUwcBEKkNy1fxhrLRPQVLjxz37885kA/VtWpm19A5vQ8A7w=
```

Comment:
Was this key intended to be committed?

Reply:
yeah, this is encrypted based on https://docs.travis-ci.com/user/encryption-keys/